Rdmp.Dicom/Cache/Pipeline/PACSSource.cs

```diff
@@ -67,11 +67,16 @@ public class PACSSource : CacheSource<SMIDataChunk>
         [DemandsInitialization("Ignore whitelist of patient identifiers",defaultValue: false, mandatory: true)]
         public bool IgnoreWhiteList { get; set; }
 
-        [DemandsInitialization("Set the DICOM priority (recommended value for NHS transfers is low)", defaultValue: DicomPriority.Low, mandatory: true)]
-        public DicomPriority Priority { get; set; }
+        [DemandsInitialization("Maximum number of times to re-request a Study when a Failure is encountered",defaultValue:3 , mandatory: true)]
+        public int MaxRetries { get; set; } = 3;
 
         private HashSet<string> _whitelist;
 
+        /// <summary>
+        /// The maximum number of tries to fetch a given Study from the PACS.  Note that retries requests might not be issued immediately after
```

Comment:
Is this comment in the right place? Looks like it might belong 4-5 lines further up, before MaxRetries is declared?

Reply:
Good spot, fixed it.